libs/@local/hash-isomorphic-utils/src/entity-store-plugin.ts

```diff
@@ -209,7 +212,7 @@ const setBlockChildEntity = (
       // updatedAt: targetEntity.updatedAt,
     };
 
-    draftEntityStore[targetEntityDraftId] = targetDraftEntity;
+    draftEntityStore[targetEntityDraftId] = castDraft(targetDraftEntity);
```

Comment:
Was getting more stranger errors about `T` not being assignable to `WritableDraft<T>` again. But I discovered some utilities in the immer docs that _seem_ to get rid of this. https://immerjs.github.io/immer/typescript/#cast-utilities